src/coreclr/jit/instrsxarch.h

```diff
@@ -611,7 +611,12 @@ INST3(kmovw_msk,         "kmovw",           IUM_WR, PCKFLT(0x91), BAD_CODE,
 INST3(kmovd_msk,         "kmovd",           IUM_WR, PCKDBL(0x91), BAD_CODE,     PCKDBL(0x90),                            INS_TT_NONE,                                          REX_W1       | Encoding_VEX)
 INST3(kmovq_msk,         "kmovq",           IUM_WR, PCKFLT(0x91), BAD_CODE,     PCKFLT(0x90),                            INS_TT_NONE,                                          REX_W1       | Encoding_VEX)
 
-INST3(LAST_K_INSTRUCTION, "LAST_K_INSTRUCTION", IUM_WR, BAD_CODE, BAD_CODE, BAD_CODE, INS_TT_NONE, INS_FLAGS_None)
+INST3(kortestb,          "kortestb",        IUM_WR, BAD_CODE,     BAD_CODE,     PACK3(0x66, 0x0F, 0x98),                 INS_TT_NONE,                                          REX_W0       | Encoding_VEX | Writes_ZF | Writes_CF ) 
+INST3(kortestw,          "kortestw",        IUM_WR, BAD_CODE,     BAD_CODE,     PACK2(0x0F, 0x98),                       INS_TT_NONE,                                          REX_W0       | Encoding_VEX | Writes_ZF | Writes_CF ) 
+INST3(kortestd,          "kortestd",        IUM_WR, BAD_CODE,     BAD_CODE,     PACK3(0x66, 0x0F, 0x98),                 INS_TT_NONE,                                          REX_W1       | Encoding_VEX | Writes_ZF | Writes_CF ) 
+INST3(kortestq,          "kortestq",        IUM_WR, BAD_CODE,     BAD_CODE,     PACK2(0x0F, 0x98),                       INS_TT_NONE,                                          REX_W1       | Encoding_VEX | Writes_ZF | Writes_CF ) 
```

Comment:
ZF is written
CF is written
OF, SF, AF, and PF should be `Resets_*`

Reply:
Done